File: Easy/BitPositions.java
import java.io.*;
public class Main {
    public static String solve(String input) {
        String[] args = input.split(",");
        Integer n = Integer.parseInt(args[0]);
        Integer p1 = Integer.parseInt(args[1]);
        Integer p2 = Integer.parseInt(args[2]);

        Integer b1 = getBitValue(n, p1);
        Integer b2 = getBitValue(n, p2);
        return Boolean.toString(b1.equals(b2));
    }

    private static Integer getBitValue(Integer number, Integer bitNumber) {
        for(Integer i = 1; i < bitNumber; i++) {
            number /= 2;
        }

        return number % 2;
    }
    
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            System.out.println(solve(line));
        }
    }
}

File: Easy/BitPositions.cpp
#include <iostream>
#include <fstream>
#include <sstream>
#include <algorithm>

using namespace std;

int getBitValue(int number, int bitNumber) {
    return (number >> (bitNumber - 1)) & 1;
}

string solve(string &input) {
    replace(input.begin(), input.end(), ',', ' ');
    istringstream input_stream(input);
    int n, p1, p2;
    input_stream >> n >> p1 >> p2;

    int b1 = getBitValue(n, p1);
    int b2 = getBitValue(n, p2);
    return (b1 == b2) ? "true" : "false";
}

int main(int argc, const char * argv[]) {
    ifstream input_stream(argv[1]);
    string input;
    
    while(getline(input_stream, input)) {
        cout << solve(input) << endl;
    }
    
    return 0;
}
